modules/check_in/config/locales/exceptions.en.yml

```diff
@@ -0,0 +1,28 @@
+en:
+  common:
+    exceptions:
+      defaults: &defaults
+        # title: required always
+        # detail: optional, if not provided it will default to title
+        # code: required always, must be unique to a specific title - no two code and titles can overlap
+        # status: require always, corresponds to the HTTP Status code
+        # links: optional, could contain an 'about' link to further details
+        # source: optional, an object containing references to the source of the error (pointer to json in validation)
+        # meta: optional, contains information suitable for debugging, VISIBLE IN PRODUCTION
+        # sentry_type: optional, defaults to 'error', supported values = debug, info, warn, error, fatal, and 'none'
+        detail: ~
+        links: ~
+        source: ~
+        meta: ~
+        sentry_type: 'error'
+      backend_service_exception: &external_defaults
+        <<: *defaults
+        title: 'Operation failed' # To be used when no external minor code is mappable
+        code: 'VA900'
+        status: 400
+      LOROTA-MAPPED-API_401:
+        <<: *external_defaults
+        title: Authentication Error
+        code: 'LOROTA-API_401'
+        detail: 'Authentication Error'
```

Comment:
Instead of an `error`, can this be marked as `debug` in sentry?

Reply:
can you please elaborate what you mean here?

Reply:
Based on the newly defined config
```
      LOROTA-MAPPED-API_401:
        <<: *external_defaults
        title: Authentication Error
        code: 'LOROTA-API_401'
        detail: 'Authentication Error'
```
With `external_defaults` inheriting `defaults` and `defaults` setting `sentry_type` as `'error'`, we seem to be logging these as `error`. Since this is 401, thinking we can just reduce it to debug level.

Reply:
we shouldn’t. this is an error from downstream, which results in vets-api returning an error (401) and needs to be logged as such